.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!--
+Please give the pull request a short but descriptive title.
+
+Use this template as a guide. Omit sections that don't apply. You may link to information rather than copy it.
+
+**edX Internal Developers:** Please also follow these steps to integrate your changes https://openedx.atlassian.net/wiki/spaces/RS/pages/1835106870/How+to+contribute+to+our+repositories
```

Comment:
Looks like this is a duplicate of https://github.com/edx/frontend-app-payment/pull/476/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R38? Should we get rid of one?

Reply:
The duplication is deliberate; the ticket is specifically "address issue of commits being merged and not deployed to production" and, while I want to make the PR template apply to internal and external developers, I don't want that message to get lost (@dianekaplan's feedback was "Can we beat them over the head with it?" 😄 )

Reply:
Sure, just keep in mind that the longer + more requests you have the more likely people's eyes will glaze over/miss stuff/skip it entirely

Reply:
I see your point about potential confusion